### PR TITLE
Update iron-autogrow-textarea.html

### PR DESCRIPTION
**Pull request by kasperpeulen**

### CODE REVIEW

File: iron-autogrow-textarea.html [hunk 1]
```diff
@@ -202,6 +202,7 @@
 
     /**
      * Returns the underlying textarea.
+     * @type HTMLTextAreaElement
```
Comment (notwaldorf) [inline iron-autogrow-textarea.html]: I think what we've been using is something like `@return {HTMLTextAreaElement} the underlying textarea`. `@type` has generally been used for properties

**Comment by notwaldorf** [comment]
Thanks for the contribution! I don't think `@type` is the right annotation, though :)

File: iron-autogrow-textarea.html (hunk 1 above)
Comment (kasperpeulen) [inline iron-autogrow-textarea.html]: @notwaldorf As far as I can see, this is the notation used for all getters in https://github.com/PolymerElements.

For example here:
https://github.com/PolymerElements/paper-header-panel/blob/master/paper-header-panel.html#L268

or here:
https://github.com/PolymerElements/iron-selector/blob/master/iron-selectable.html#L147

Comment (notwaldorf) [inline iron-autogrow-textarea.html]: Ah, I think you're right.

**Pull request merged**